### PR TITLE
Fix python install

### PR DESCRIPTION
**Pull request by twicki**
## Technical Description

Fixes the python installation

### Resolves

Broken master with disabled tests

### Testing

See gt4py2dawn

### CODE REVIEW

File: dawn/setup.py [hunk 1]
```diff
@@ -102,20 +105,21 @@ def run(self):
...
+        build_args = ["--config", cfg, "-j", str(BUILD_JOBS)]
         cmake_cmd = ["cmake", "--build", "."] + build_args
-        print("{cwd} $ {cmd}".format(cwd=self.bundle_build_temp, cmd=" ".join(cmake_cmd)))
-        subprocess.check_call(cmake_cmd, cwd=self.bundle_build_temp)
+        print("{cwd} $ {cmd}".format(cwd=self.build_tmp, cmd=" ".join(cmake_cmd)))
+        subprocess.check_call(cmake_cmd, cwd=self.build_tmp)
 
         # Move from build temp to final position
         for ext in self.extensions:
+            print(ext)
```
Comment (mroethlin) [inline dawn/setup.py]: remove plz

File: dawn/setup.py [hunk 2]
```diff
@@ -70,30 +73,30 @@ def run(self):
...
-                if str(self.bundle_build_temp) != cache_build_dir:
-                    shutil.rmtree(self.bundle_build_temp, ignore_errors=False)
-                    shutil.rmtree(os.path.join(BUNDLE_ABS_DIR, "install"), ignore_errors=True)
+                if str(self.build_tmp) != cache_build_dir:
+                    shutil.rmtree(self.build_tmp, ignore_errors=False)
+                    shutil.rmtree(os.path.join(DAWN_DIR, "install"), ignore_errors=True)
                     assert not os.path.exists(cmake_cache_file)
-        os.makedirs(self.bundle_build_temp, exist_ok=True)
+        os.makedirs(self.build_tmp, exist_ok=True)
+        os.makedirs(os.path.join(DAWN_DIR, "install"), exist_ok=True)
```
Comment (jdahm) [inline dawn/setup.py]: Why does this use `DAWN_DIR` but `build_tmp` uses `DAWN_ABS_DIR`?

File: dawn/setup.py [hunk 3]
```diff
@@ -70,30 +73,30 @@ def run(self):
         # Installing in editable/develop mode builds the extension in the original build path,
         # but a regular `pip install` copies the full tree to a temporary folder
         # before building, which makes CMake fail if a CMake cache had been already generated.
-        self.bundle_build_temp = str(os.path.join(BUNDLE_ABS_DIR, "build"))
-        cmake_cache_file = os.path.join(self.bundle_build_temp, "CMakeCache.txt")
+        # self.bundle_build_tmp = str(os.path.join(BUNDLE_ABS_DIR, "build"))
```
Comment (jdahm) [inline dawn/setup.py]: Remove comment before merging.

File: dawn/setup.py (hunk 2 above)
Comment (twicki) [reply dawn/setup.py]: oversight

**Comment by twicki** [comment]
launch jenkins

File: dawn/setup.py (hunk 1 above)
Comment (twicki) [reply dawn/setup.py]: done

File: dawn/setup.py (hunk 3 above)
Comment (twicki) [reply dawn/setup.py]: done

**Pull request merged**